swagger.yaml

```diff
@@ -244,6 +245,101 @@ paths:
           description: Wrong content type (should be JSON)
         422:
           description: Missing paramaters
+  /projects/{projectId}/resources:
+    parameters:
+      - in: path
+        name: projectId
+        schema:
+          type: integer
+        required: true
+    get:
+      tags:
+        - resources
+        - projects
+      summary: Gets metadata of resources in project
+      security:
+        - user_id: []
+      description: Returns a list of metadata of all the resources in the given project
+      responses:
+        200:
+          description: Resource created successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ResourceMetadata"
+        400:
+          description: Malformed Request
+        404:
+          description: Project not found
+        422:
+          description: Missing paramaters
+    post:
+      tags:
+        - resources
+        - projects
+      summary: Creates new resource in project
+      security:
+        - user_id: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ResourceMetadata"
+      description: Creates a new resource with given name for the given project
+      responses:
+        200:
+          description: Resource created successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Resource"
+        400:
+          description: Malformed Request
+        404:
+          description: Project not found
+        409:
+          description: Duplicate resource
+        415:
+          description: Wrong content type (should be JSON)
+        422:
+          description: Missing paramaters
+  /projects/{projectId}/resources/{resourceId}:
+    parameters:
+      - in: path
+        name: projectId
+        schema:
+          type: integer
+        required: true
+      - in: path
+        name: resourceId
+        schema:
+          type: integer
+        required: true
+    put:
+      tags:
+        - resources
+        - projects
+      summary: Updates resource content in project
+      security:
+        - user_id: []
+      requestBody:
+        content:
+          image/*:
+            schema:
+              type: string
+              format: binary
+      description: Uploads new content of a given resource in a given project
+      responses:
+        204:
+          description: Resource uploaded successfully
+        400:
+          description: Malformed Request
+        404:
+          description: Project or resource not found
+        422:
+          description: Missing paramaters
```

Comment:
What about limits for image size or any kind of validation?

Reply:
done